src/System.Linq/tests/ExceptTests.cs

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Linq.Tests
+{
+    public class ExceptTests
+    {
+        // Class which is passed as an argument for EqualityComparer
+        private class AnagramEqualityComparer : IEqualityComparer<string>
```

Comment:
I've seen this class copied and used in a bunch of the unit tests by the legay tests, might be a good future cleanup to extract it and use the same one everywhere (assuming it *is* actually identical).